File: 프로그래머스/0/120889. 삼각형의 완성조건 （1）/삼각형의 완성조건 （1）.java
import java.util.*;

class Solution {
    public int solution(int[] sides) {
        int answer = 0;
        Arrays.sort(sides);
        
        if(sides[2] < (sides[0] + sides[1])) {
            answer = 1;
        } else answer = 2;
        
        return answer;
    }
}

File: 프로그래머스/0/120889. 삼각형의 완성조건 （1）/삼각형의 완성조건 （1）.js
function solution(sides) {
    // 배열을 오름차순 정렬
    sides.sort((a, b) => a - b);

    // 가장 긴 변이 나머지 두 변의 합보다 작은지 확인
    if (sides[2] < sides[0] + sides[1]) {
        return 1; // 삼각형 가능
    } else {
        return 2; // 삼각형 불가능
    }
}
